docs/source/tutorials_and_examples/events/A-voice-operated-elevator-using-events.md

```diff
@@ -1,8 +1,5 @@
 # A voice operated elevator using events
 
-
-- Previous tutorial: [Adding dialogues in events](Dialogues-in-events)
```

Comment:
All this events-related stuff (this and the dialogues) is part of the In-Game-Python contrib, so it belongs under Contribs.